tests/utils/verification/volumeproperties.go

```diff
@@ -77,7 +78,7 @@ func GetVolumePropertiesAdminCli(volName string, hostname string) string {
 // for volume using Docker cli
 func GetVolumePropertiesDockerCli(volName string, hostname string) string {
 	cmd := dockercli.InspectVolume + " --format '{{index .Status.capacity.size}} {{index .Status.diskformat}} {{index .Status \"attached to VM\"}}' " + volName
-	op := ExecCmd(hostname, cmd)
+	op, _ := ssh.InvokeCommand(hostname, cmd)
 	expctedLen := 0
 	if op == "" {
```

Comment:
All instances of log.Fatal should be gocheck Assert() whcih ensures that only the current test is failed.